pkg/reporter/json_report.go

```diff
@@ -257,6 +263,15 @@ func (j *jsonReportGenerator) buildJsonPackageReportFromPackage(p *models.Packag
 			Aliases:    utils.SafelyGetValue(vuln.Aliases),
 			Severities: severties,
 		})
+
+		if (len(pkg.Vulnerabilities) > 0 && flag) {
+			pkg.Advices = append(pkg.Advices, &schema.RemediationAdvice{
+				TargetAlternatePackageVersion:	updateToVersion,
```

Comment:
You should also set the `Type` field of the `RemediationAdvice` struct

```
Type: schema.RemediationAdviceType_UpgradePackage
```

Reply:
Done :)